qa/L0_infer/test.sh

```diff
@@ -96,6 +96,7 @@ else
     TRITON_DIR=${TRITON_DIR:="/opt/tritonserver"}
     SERVER=${TRITON_DIR}/bin/tritonserver
     BACKEND_DIR=${TRITON_DIR}/backends
+    export SERVER_LD_PRELOAD=/usr/lib/$(uname -m)-linux-gnu/libgomp.so.1
```

Comment:
Why is this line required?

Reply:
Hi @Tabrizian  , 
Thank you for asking over provided change. 
We are using this variable to preload specific library, as you remember we had done similar job on MA side: 
https://github.com/triton-inference-server/model_analyzer/pull/543/files

Reply:
So this is required only for PyTorch models? The L0_infer models are fairly simple, that is why I was wondering why it is required.